tests/end-to-end/api/01-users.js

```diff
@@ -16,6 +16,47 @@ import { customFieldText, clearCustomFields, setCustomFields } from '../../data/
 import { updatePermission, updateSetting } from '../../data/permissions.helper';
 import { createUser, login } from '../../data/users.helper.js';
 
+function createTestUser() {
```

Comment:
We already have a function to create tests users as you can see [here](https://github.com/RocketChat/Rocket.Chat/blob/develop/tests/data/users.helper.js#L4).

Reply:
I use the helper code now